.github/workflows/ci.yml

```diff
@@ -0,0 +1,194 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        fetch-depth: 1
+    - uses: actions/setup-node@v2-beta
+      with:
+        node-version: '10.x'
+    - name: Get package manager's global cache path
+      id: global-cache-dir-path
+      run: echo "::set-output name=dir::$(yarn cache dir)"
+    - name: Cache package manager's global cache
+      id: cache-global-package-manager-cache
+      uses: actions/cache@v1
+      with:
+        path: ${{ steps.global-cache-dir-path.outputs.dir }}
+        key: ${{ runner.os }}-${{ matrix.node-version }}-yarn-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-yarn-
+    - name: Cache node_modules
+      id: cache-node-modules
+      uses: actions/cache@v1
+      with:
+        path: node_modules
+        key: ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-${{ hashFiles('**/yarn.lock') }}
+        restore-keys: |
+          ${{ runner.os }}-${{ matrix.node-version }}-nodemodules-
+    - name: Install Dependencies
+      run: yarn install --frozen-lockfile
+      if: |
+        steps.cache-global-package-manager-cache.outputs.cache-hit != 'true' ||
+        steps.cache-node-modules.outputs.cache-hit != 'true'
+    - name: Lint
+      run: yarn lint
+
+  test:
+    name: Tests
+    runs-on: ${{ matrix.os }}
```

Comment:
This may have changed now. When I wrote workflows around March, I realized that we couldn't customize the output job name if we use `strategy.matrix`.

For example, in the Conversation tab, you will see the name `CI / Tests (ubuntu-latest, chrome)` with `ubuntu-latest` and `chrome` inside parentheses.

<img width="800" alt="Screen Shot 2020-11-28 at 10 34 09 AM" src="https://user-images.githubusercontent.com/16869656/100520778-468c7480-3165-11eb-9a1f-9f35b52c7381.png">

If you prefer to keep the names simple, we can just write `runs-on: ubuntu-latest`.

https://gist.github.com/ijlee2/25004d700cd008ee08ecc4873c109a5b#file-ci-yml-L137

Reply:
I liked that flexible OS matrix because it _should_ make it easier to customize the CI workflow to also run the tests against Safari on iOS. But not sure if that would work out in practice at all. I see that hard-coding `ubuntu-latest` would simplify.